.github/workflows/build-images.yml

```diff
@@ -360,7 +360,14 @@ jobs:
         # pushing it up to an external repository.
         if [ $GITHUB_EVENT_NAME != 'pull_request' ]
         then
-          make docker-build-crossplatform-operator
+          docker manifest inspect ${OPERATOR_IMG} > /dev/null 2>&1
+          EXIT_CODE=$?
+          if [ $EXIT_CODE -eq 0 ]; then
+            echo "$image exists"
+          else 
+            echo "$image does not exist, making a push"
+            make docker-build-crossplatform-operator
```

Comment:
So we are not going to push to github repo for non-pr cases?

Reply:
By existing logic, yes. If we need to change that I can do it in this PR as well.